behat.yml.dist

```diff
@@ -7,6 +7,7 @@ default:
         - Drupal\DrupalExtension\Context\DrupalContext
         - Drupal\DrupalExtension\Context\MessageContext
         - Drupal\DrupalExtension\Context\MinkContext
+        - Drupal\Tests\oe_media\Behat\MediaContext
```

Comment:
I am not 100% sure we need to import this. We only use it in for one step (as far as I can tell) : "Given the following image"
Now we could simply create the image manually going through the forms (indeed we already do that on another feature), but it might be better to use this context and simplify the tests here instead. I'm leaving this comment so coordination team can give their opinion.

Reply:
This was asked by coordination team above so resolved.